examples/tests/valid/benchmarks/empty.w

```diff
@@ -0,0 +1,7 @@
+/*\
+cases:
+  - target: sim
+    maxMeanTime: 900
+  - target: tf-aws
+    maxMeanTime: 1000
+\*/
```

Comment:
This comment style is a callback to one of the original versions of the spec, so I thought it was a fun homage. I'm definitely open to feedback on both the comment style and the data structure what's inside it.